### DISCUSSION TITLE
零代碼爬蟲神器

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#538] [General]
### Web Scraper
我們常常會遇到一些簡單的需求，需要爬取某網站上的一些數據，但這些頁面的結構非常簡單，且數據量比較小，自己寫代碼固然可以實現，但有更方便的工具讓你不用那麼麻煩，Web Scraper，它是Chrome瀏覽器的一個擴展插件，安裝後你可以直接在F12調試工具中使用它。
教學:
(1)https://b0444135.medium.com/web-web-scraper教學-輕鬆爬網頁-811899645405
(2)https://blog.csdn.net/weixin_42968145/article/details/116105263

**Comment by awkward-willy** [comment]
透過簡單的 extensions 就可以快速獲取需要的資訊，在小規模的使用上感覺真的相當方便！

**Comment by BingRong1217** [comment]
感覺是很方便很實用的工具！

**Comment by Ben6uo** [comment]
感覺這個東西在日後的程式上有超大幫助

**Comment by williamyen0623** [comment]
如果是要做小規模的資料爬蟲，那麼上述的工具用起來肯定會更加方便，但自己寫代碼的好處就是可以執行更加細緻的客製化

**Comment by HsuChen88** [comment]
簡易的爬蟲用這個工具的確方便，看來是時候重操舊業了。

**Comment by Raywei0214** [comment]
說到網路爬蟲，你會想到什麼呢？話雖如此，我們卻也不能夠這麼篤定。看看別人，再想想自己，會發現問題的核心其實就在你身旁。
